### DISCUSSION TITLE
INCLASS_2

### DISCUSSION THREAD

**Discussion by gdlc** [gdlc/STAT_COMP#3] [General]
Ask question, post tricks and tips, answer questions that other post. Most importantly, be respectful.

**Comment by hepkerga** [comment]
Hello, I am working on the in-class 3 assignment, and I am having a hard time trying to store the outputs of the for loops as a vector. For the example below, I tried to assign the Q1.1 to the print(x) outputs of the for loops and store it as a vector named Q1.1. However, it only shows the fourth output value of the for loop when printing out the Q1.1 vector.  When possible, please let me know of potential solutions on how to print out all outputs of a for loop and to store as a vector. Thank you.

**Reply by gdlc** [reply]
Hi, i just created an entry for INCLASS-3 and copied your question there

Thanks!
